README.md

```diff
@@ -15,6 +16,9 @@ A basic set of tests can be run from [`scripts/test`](scripts/test)
 The data in a running instance can be queried using the ElasticSearch REST API and a command line tool like curl.
 
 ## Full Text Search Example
+=======
```

Comment:
I assume this shouldn't be here? And the following 2 lines don't appear to make too much sense.